### DISCUSSION TITLE
Static image on Home instead of live display

### DISCUSSION THREAD

**Discussion by jmanalansan** [photobooth-app/photobooth-app#328] [Q&A]
Hello,

Is it possible to disable the live preview on the homepage and replace it with a static display image or photo? Ideally, the live preview or display should only appear once the “Take a Photo” button is clicked.

Thank you!

**Comment by mgineer85** [comment]
Yes, you can disable live preview and modify the ui setting frontpage text. You can set any html freely and use uploaded files in the userdata folder.
